api/src/main/java/org/pmiops/workbench/db/model/DbInstitutionTierRequirement.java

```diff
@@ -0,0 +1,111 @@
+package org.pmiops.workbench.db.model;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "institution_tier_requirement")
+public class DbInstitutionTierRequirement {
+
+  private long institutionTierRequirementId;
+  private DbInstitution institution;
+  private DbAccessTier accessTier;
+  private boolean eraRequired;
+  private RequirementEnum requirementEnum;
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @Column(name = "institution_tier_requirement_id", nullable = false)
+  public long getInstitutionTierRequirementId() {
+    return institutionTierRequirementId;
+  }
+
+  public DbInstitutionTierRequirement setInstitutionTierRequirementId(
+      long institutionTierRequirementId) {
+    this.institutionTierRequirementId = institutionTierRequirementId;
+    return this;
+  }
+
+  @ManyToOne
+  @JoinColumn(name = "institution_id", nullable = false)
+  public DbInstitution getInstitution() {
+    return institution;
+  }
+
+  public DbInstitutionTierRequirement setInstitution(DbInstitution institution) {
+    this.institution = institution;
+    return this;
+  }
+
+  @ManyToOne()
+  @JoinColumn(name = "access_tier_id", nullable = false)
+  public DbAccessTier getAccessTier() {
+    return accessTier;
+  }
+
+  public DbInstitutionTierRequirement setAccessTier(DbAccessTier accessTier) {
+    this.accessTier = accessTier;
+    return this;
+  }
+
+  @Column(name = "era_required")
+  public boolean getEraRequired() {
+    return eraRequired;
+  }
+
+  public DbInstitutionTierRequirement setEraRequired(boolean eraRequired) {
+    this.eraRequired = eraRequired;
+    return this;
+  }
+
+  @Enumerated(EnumType.STRING)
+  @Column(name = "requirement_enum", nullable = false)
+  public RequirementEnum getRequirementEnum() {
```

Comment:
So this is using the enum type I gather - what is the behavior if we typo one of these enums, or it doesn't match the set of values defined in the database schema?

Reply:
surprisingly, I tried with some random name, and it does insert into DB successfully with my string...

Reply:
OK, would be good to gather up these caveats and behaviors so we can make a decision about migrating to this pattern or not